docs/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM alpine:3.8@sha256:621c2f39f8133acb8e64023a94dbdf0d5ca81896102b9e57c0dc184cadaf5528
```

Comment:
Do we need a particular SHA version? Can we remove it?

Reply:
Sorry it's just habit. If we trust alpine, which we probably should as it's an official build then not locking it down to the digest is fine. I'll get it removed.